vochain/start.go

```diff
@@ -23,9 +23,14 @@ import (
 	tmjson "github.com/tendermint/tendermint/libs/json"
 	tmlog "github.com/tendermint/tendermint/libs/log"
 	tmos "github.com/tendermint/tendermint/libs/os"
-	"go.vocdoni.io/dvote/log"
+	logger "go.vocdoni.io/dvote/log"
```

Comment:
if you call `Named` directly in the global var decl, then we don't need these renames.

Reply:
mmmh, i tried that first, as in 
`var log = log.Logger()`
or, well, the ideal `var log = log.Named("ipfssync")`
but go complains that `log already declared through import of package log ("go.vocdoni.io/dvote/log")`

Reply:
Ah, I guess because of the global scope. then maybe `var log = logpkg.Named("ipfssync")`